scripts/spec_xml/paths.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from enum import Enum
+
+# Define a branch enum for different versions or branches
+
+
+class Branch(Enum):
+    MASTER = "master"
+    IN_PROGRESS = "in_progress"
+    V1_3 = "v1_3"
+
```

Comment:
The V1_4 entry will need to be added in the near future, so I'd create an issue for that.

Reply:
Marking resolved: should probably be done as a aseparate PR.